Gemfile

```diff
@@ -5,7 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-ruby '2.4.0'
+ruby '2.4.2'
```

Comment:
Whilst we're doing this can we change it to `ruby File.read(".ruby-version").chomp`